ERC721A.sol

```diff
@@ -325,6 +327,25 @@ contract ERC721A is
   function _exists(uint256 tokenId) internal view returns (bool) {
     return tokenId < currentIndex;
   }
+  
+  function _safeBurn(uint256 tokenId) internal {
+    _safeBurn(tokenId);
```

Comment:
x

Reply:
yeah _safeBurn is covered with implementation